### ISSUE TITLE
Preset option doesn't work

### ISSUE THREAD

**Issue by MCluck90** [AtomLinter/linter-jscs#41]
I just installed linter and linter-jscs then set the preset to "crockford". No errors were reported for this snippet:

```javascript
for (var i = 0; i < 10; i++)
    console.log(i);
```

even though the Crockford preset requires curly braces around all `for`s. I then created a `.jscsrc` file containing this:

```json
{
    "preset": "crockford"
}
```

and the correct error was reported.

**Specs:**

  * Windows 8.1
  * Atom v0.188.0
  * linter v0.12.0
  * linter-jscs v1.9.0

**Comment by iam4x** [comment]
@MCluck90 do you have any `.jscsrc` in the parent directories? Or in your home dir?

**Comment by MCluck90** [comment]
Oh man, I guess when I was trying out JSCS months ago I forgot I created a .jscsrc in my home directory. Removing it fixed the problem. Thanks @iam4x .
